backend/internal/model/game.go

```diff
@@ -32,6 +34,13 @@ type GameScore struct {
 	WhiteScore int `json:"whiteScore"`
 }
 
+type ManualGoals struct {
```

Comment:
Why is this structure "flat"? Now you have to support them in a special way using switches. 

Have you thought about using something like a two-dimensional array (or map) with [teamId][added/subtracted]? IMO, in this case, access to data is much simpler.

Reply:
I think it is simple enough for us and at least for now we will leave it as it is.

Reply:
OK, but note that now  `UpdateManualGoals`, `addManualGoal`, `subManualGoal` are doing something like `manualGoals[teamId][add/sub]++`. These functions are about 30 lines of code. Each line is a cost related to its maintenance in the future :)

Reply:
We discussed this idea and replaced our structure with two-dimensional map. Could you check if our changes are correct?

Reply:
Wow, very good!

Reply:
Okay, thank you for the advice :)